### ISSUE TITLE
Bug report , args.learning_rate is not used

### ISSUE THREAD

**Issue by sharpstill** [labsix/limited-blackbox-attacks#15]
The `current_lr` of line https://github.com/labsix/limited-blackbox-attacks/blob/master/attacks.py#L51 is replaced by `max_lr` of line https://github.com/labsix/limited-blackbox-attacks/blob/master/attacks.py#L219
It means the `args.learning_rate` is totally not used, why?

**Comment by jlin816** [comment]
Thanks for catching this! `args.learning_rate` is extraneous and should have been deleted. We implemented backtracking line search for the learning rate (with `min_lr` and `max_lr` as parameters) so we could make fewer queries for the partial info attack.

**Comment by jlin816** [comment]
Fixed.
